### PR TITLE
Adds page on ComposeDB

### PR DESCRIPTION
**Pull request by jmcook1186**
Adds page introducing ComposeDB.

Took me a while to get my head around - maybe there's more detail required on the Passport schema?
Raising this PR anyway to start discussion.

### CODE REVIEW

File: pages/building-with-passport/passport-api/composedb.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+# ComposeDB
+
+ComposeDB is a decentralized graph database built on top of ceramic.
```
Comment (lebraat) [inline pages/building-with-passport/passport-api/composedb.md:3]: capitalize Ceramic

File: pages/building-with-passport/passport-api/_meta.json [hunk 2]
```diff
@@ -4,6 +4,7 @@
     "quick-start-guide": "Quick start",
     "tutorials": "Tutorials",
     "api-reference": "API reference",
+    "composedb": "ComposeDB",
```
Comment (lebraat) [inline pages/building-with-passport/passport-api/_meta.json:7]: Let's actually move this out to the same level as Passport API and Smart Contract, as we'll be marketing it as a uniquely different from the API.

File: pages/building-with-passport/passport-api/composedb.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+Being a product built on top of Ceramic, ComposeDB provides many benefits for web3 native applications including strong Ethereum integrations. There is also a rich, Graph-QL style API that helps developers to construct complex queries. These are very useful features for Gitcoin Passport.
+
+Perhaps more importantly, ComposeDB allows us to create and publish data schemas that work across different tech-stacks, meaning we can deploy apps that work identically across multiple layer-1 blockchains and their layer-2's. Data on ComposeDB is updateable allowing, for example, Stamp expiry and reverification.
+
+Read more in the [ComposeDB documentation](https://developers.ceramic.network/docs/composedb/getting-started).
+
+## Passport on ComposeDB
+
+The data used by Gitcoin passport fits very well into the graph structure. There are two basic elements that Passport considers: user addresses and Stamps. These are nodes. The links between Stamps and users and edges.
```
Comment (lebraat) [inline pages/building-with-passport/passport-api/composedb.md:18]: "Passport" capitalize

**Review by lebraat — COMMENTED** [review]
Looks great! Just a few suggested updates. 

I'll be wrapping my head around composeDB more in the coming weeks and might have some suggested updates then, but for now, this is a great foundation.

File: pages/building-with-passport/passport-api/composedb.md (hunk 1 above)
Comment (jmcook1186) [reply pages/building-with-passport/passport-api/composedb.md:3]:
```suggestion
ComposeDB is a decentralized graph database built on top of Ceramic.
```

File: pages/building-with-passport/passport-api/composedb.md (hunk 3 above)
Comment (jmcook1186) [reply pages/building-with-passport/passport-api/composedb.md:18]:
```suggestion
The data used by Gitcoin Passport fits very well into the graph structure. There are two basic elements that Passport considers: user addresses and Stamps. These are nodes. The links between Stamps and users and edges. 
```

**Pull request merged**